### DISCUSSION TITLE
More Abdl dialogue as hearts go up?

### DISCUSSION THREAD

**Discussion by Bloopl** [zippity21/Stardew_Valley_Regression_Mod#95] [General]
I've just recently started olaying with this mod, loving it so far! I got the little notif saying that as hearts go up more abdl related dialogue will be unclocked in regards to Jas and Vincent, but is this true for the other villagers as well?

Also I'm curious, are there any thoughts on a pacifiwr item being added in the future? I think that'd be really neat! ^^

**Comment by Gooberology** [comment]
Srry for the late reply. Yeah, it does apply to other villagers as well. Villagers either open up to you about their own abdl related tendencies, come to accept your own, or start wearing diapers themselves. 

I've been trying to start adding clothing options in the next update, but thatd be a content patcher mod, and im having trouble finding some documentation on it. That and im bouncing between several projects at the moment. Id love to add it but we'll just have to see.
